android/activation/pom.xml

```diff
@@ -39,7 +39,7 @@
     <properties>
 	<activation.spec.version>1.2</activation.spec.version>
 	<activation.extensionName>
-	    javax.activation
+	    jakarta.activation.android
```

Comment:
I see apart from replacing `javax` with `jakarta`, `android` too has been appended to the extension name. I hope this is as expected?

Reply:
Yes, I want to distinguish this special Android version from the real JAF version.